Engine/Settings.cs

```diff
@@ -31,8 +31,16 @@ public class Settings
         private List<string> customRulePath;
         private Dictionary<string, Dictionary<string, object>> ruleArguments;
 
-        public bool RecurseCustomRulePath => recurseCustomRulePath;
-        public bool IncludeDefaultRules => includeDefaultRules;
+        public bool RecurseCustomRulePath
```

Comment:
The modern way to do this would be
```csharp
public bool RecurseCustomRulePath { get; }
```

Reply:
I needed to change these to settable, so I went the way of properties.

Reply:
Then one would do this:
```csharp
public bool RecurseCustomRulePath { get; set; }
```